cpp-client-reference/src/volition_shim.cpp

```diff
@@ -0,0 +1,68 @@
+#include "volition.h"
+
+struct vl_optimizer_s {};
```

Comment:
what is this supposed to encapsulate?

Reply:
This concrete `struct` will encapsulate Protobuf types (channel, optimizer stub, etc.).